src/System.Private.CoreLib/shared/System/Threading/SemaphoreSlim.cs

```diff
@@ -578,7 +565,7 @@ public Task<bool> WaitAsync(int millisecondsTimeout)
         /// </exception>
         /// <exception cref="T:System.ArgumentOutOfRangeException">
         /// <paramref name="timeout"/> is a negative number other than -1 milliseconds, which represents 
-        /// an infinite time-out -or- timeout is greater than <see cref="System.int.MaxValue"/>.
```

Comment:
Nit: The original version of this comment looked right. The new version does not make sense to me.